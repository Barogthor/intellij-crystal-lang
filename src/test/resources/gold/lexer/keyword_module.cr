module Curses
  class Window
  end
end

Curses::Window.new