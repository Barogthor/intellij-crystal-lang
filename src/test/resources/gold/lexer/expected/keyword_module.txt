module ('module')
WHITE_SPACE (' ')
STRING ('C')
IDENT ('urses')
WHITE_SPACE ('\n  ')
class ('class')
WHITE_SPACE (' ')
STRING ('W')
IDENT ('indow')
WHITE_SPACE ('\n  ')
end ('end')
WHITE_SPACE ('\n')
end ('end')
WHITE_SPACE ('\n\n')
STRING ('C')
IDENT ('urses')
:: ('::')
STRING ('W')
IDENT ('indow')
. ('.')
new ('new')